demo/app.py

```diff
@@ -86,59 +131,64 @@ def load_text_to_speech_model_and_tokenizer():
         )
 
     st.divider()
-    st.header("Downloading and Loading models")
-    st.markdown(
-        "[Docs for this Step](https://mozilla-ai.github.io/document-to-podcast/step-by-step-guide/#step-2-podcast-script-generation)"
-    )
-    st.divider()
+    text_model = load_text_to_text_model()
 
-    st.markdown(
-        "For this demo, we are using the following models: \n"
-        "- [OLMoE-1B-7B-0924-Instruct-GGUF](https://huggingface.co/allenai/OLMoE-1B-7B-0924-Instruct-GGUF)\n"
-        "- [parler-tts-mini-v1](https://huggingface.co/parler-tts/parler-tts-mini-v1)"
-    )
-    st.markdown(
-        "You can check the [Customization Guide](https://mozilla-ai.github.io/document-to-podcast/customization/)"
-        " for more information on how to use different models."
+    model_name = st.selectbox(
```

Comment:
@Kostis-S-Z Unless I misunderstood, I thought we agree on picking 1 model for the demo and document how to use a different one in the customization guide (using the CLI and/or direct API usage)

Reply:
Indeed thats what we said, and I dont mind re-implementing it like that. I just thought that this is more user-friendly and doesnt cost us anything :man_shrugging: . I ll implement whatever we think its better for UX

Reply:
@Kostis-S-Z  Lets go ahead with just picking 1 model for the demo - makes its easier for it to be robust with 1 model + prompt combo. @daavoo 's CLI branch should make it easier for people to switch if they want to.